src/clover/load/load.py

```diff
@@ -237,7 +248,7 @@ def from_dict(cls, device_input: Dict[str, Any]) -> Any:
 #   The default kerosene device to use in the event that no kerosene information is
 #   provided.
 DEFAULT_KEROSENE_DEVICE = Device(
-    False, DemandType.DOMESTIC, 1, 0, 0, 0, 0, KEROSENE_DEVICE_NAME, 0, 0
+    False, DemandType.DOMESTIC, 1, 0, 0, 0, 0, 0, KEROSENE_DEVICE_NAME, 0, 0
```

Comment:
## :bug: Load-time for kerosene
I think the load time for Kerosene should be `1` rather than `0`? It's annoying that it's so hard-baked, but there you go! 😉